CMakeLists.txt

```diff
@@ -74,20 +74,27 @@ if(WIN)
     ./common/utils_win.cc
     ./common/utils_win.h
   )
+  set(PLATFORM_TEST_CODE
+    ./agent/src/event_win_unittest.cc
+  )
 elseif(MAC)
   set(PLATFORM_AGENT_CODE
     ./agent/src/agent_mac.cc
     ./agent/src/agent_mac.h
     ./agent/src/event_mac.cc
     ./agent/src/event_mac.h
   )
+  set(PLATFORM_TEST_CODE
```

Comment:
Oops, I was going to check that before uploading this PR and forgot.  It seems that cmake does not like empty lists when calling add_executable() below, so I added test files for linux and mac too.  Since the implementation is not yet complete on those platforms there are fewer test cases.